scripts/modules/b2b-account/users.js

```diff
@@ -297,11 +295,11 @@ define(["modules/mozu-utilities", "modules/jquery-mozu", 'modules/api', "undersc
             var viewB2BAccount = self.model.attributes.viewB2BAccount;
             if (viewB2BAccount) {
                 collection = new B2BUsersGridCollectionModel({});
+                collection.lastRequest.accountId = self.model.attributes.accountToView;
```

Comment:
Can you add a comment here explaining why this is being done instead of using `baseRequestParams`. I would also leave a comment on the `B2BUsersGridCollectionModel` mentioning why you are not using `baseRequestParams`.

Reply:
I tried all option with baseRequestParams but the value of accountToView wasn't set to accountId in route param.
Even I didn't see any code in the entire core theme to update route parameter on the collection model. There may be another option to use baseRequestParams but I didn't find.
So while debugging I found that the request is getting generated by using lastRequest object so I updated accountId in that object and it worked. it's weird.

Reply:
Yea the code is fine. Just add a comment explaining why it's weird :)